### ISSUE TITLE
Atlas hosting for Vagrant boxes has ended

### ISSUE THREAD

**Issue by Sharpie** [Sharpie/puppet-debugging-kit#19]
Hosting of Vagrant boxes on atlas.hashicorp.com has moved to vagrantcloud.com and this functionality of atlas has been decommissioned, apparently some time last week. This means that VM creation fails when a box has to be downloaded:

```
Bringing machine 'pe-201732-master' up with 'virtualbox' provider...
==> pe-201732-master: Box 'puppetlabs/centos-7.2-64-nocm' could not be found. Attempting to find and install...
    pe-201732-master: Box Provider: virtualbox
    pe-201732-master: Box Version: >= 0
The box 'puppetlabs/centos-7.2-64-nocm' could not be found or
could not be accessed in the remote catalog. If this is a private
box on HashiCorp's Atlas, please verify you're logged in via
`vagrant login`. Also, please double-check the name. The expanded
URL and error message are shown below:

URL: ["https://atlas.hashicorp.com/puppetlabs/centos-7.2-64-nocm"]
Error: The requested URL returned error: 404 Not Found
```

Vagrant migrated to using the new service URL in 1.9.6:

https://github.com/hashicorp/vagrant/commit/5f955c3d38

Debug Kit is currently stuck on 1.8.7 due to test suite issues, so we'll need to figure out a workaround for this version. Two options:

  - Monkeypatch the `Vagrant::DEFAULT_SERVER_URL` constant to swap in `https://vagrantcloud.com` when needed.

  - Update the vms.yaml example to add a default that sets `vm.box_server_url`:

```yaml
vm_defaults:
  box_server_url: "https://vagrantcloud.com"
```

**Comment by Sharpie** [comment]
`Vagrant:: DEFAULT_SERVER_URL` is present all the way back to Vagrant 1.6 which is the last version that the Oscar test suite cares about. I'm going to go with this approach since it fixes the issue for everyone without requiring YAML configuration to be manually updated by users --- even though it is a hack compared to the config option.

**Comment by Sharpie** [comment]
Fixed by db09862.
